recipes/gget/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "gget" %}
+{% set version = "0.0.24" %}
+{% set sha256 = "c9845dbbbf08b5bfada304c495e96d9ef63574a2ba88921ef06f9e6b15f2e958" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/pachterlab/gget/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install --no-deps --ignore-installed -vv .
+
+requirements:
+  build:
+    - python >=3.6
+    - pip
+    - setuptools
+  host:
+    - python >=3.6
+    - numpy >=1.17.2
+    - pandas >=1.0.0
+    - requests >=2.22.0
+    - ipython
+    - matplotlib-base
+    - mysql-connector-python >=8.0.5
+    - beautifulsoup4 >=4.10.0
+  run:
+    - python >=3.6
+    - numpy >=1.17.2
+    - pandas >=1.0.0
+    - requests >=2.22.0
+    - ipython
+    - matplotlib-base
+    - mysql-connector-python >=8.0.5
+    - beautifulsoup4 >=4.10.0
+
+test:
+  commands:
+    - {{ name }} --help
+
+about:
+  home: "https://github.com/pachterlab/gget"
+  license: BSD 2-Clause
```

```suggestion
  license: BSD-2-Clause
```
Now it's a valid SPDX identifier.